client/app/scripts/superdesk-authoring/views/sd-template-select.html

```diff
@@ -0,0 +1,28 @@
+<div sd-modal data-model="open" class="output-channel-modal">
+    <div class="modal-header"> <a href="" class="close" ng-click="close()"><i class="icon-close-small"></i></a>
+        <h3 translate>Select template</h3>
+    </div>
+    <div class="modal-body">
+        <form>
+            <fieldset>
+                <div class="field">
+                    <label for="keyword" translate>Keyword</label>
+                    <input type="text" id="keyword" ng-model="options.keyword" ng-model-options="{ debounce: 300 }">
+                </div>
+                <div class="field">
+                    <label translate>Templates</label>
+                    <ul>
+                        <li ng-repeat="template in templates._items">
```

Comment:
Doesn't it render only a button with the `template_name` here, and not the whole template?
(hence my other comment later that `track by` here is non-essential, only recommended)